app/components/documents/document-details-panel.js

```diff
@@ -82,6 +82,8 @@ export default class DocumentsDocumentDetailsPanel extends Component {
       }
       this.args.onChangeFile();
     }
+    const now = new Date();
+    this.args.piece.created = now;
```

Comment:
@StuyckRuben I think these 2 lines belongs inside the `if (this.replacementSourceFile) `, before the `yield this.args.piece.save();` on line 73.

Likely this is also the cause of the failing test